src/main/java/com/cleanroommc/groovyscript/compat/mods/mekanism/Mekanism.java

```diff
@@ -38,43 +38,20 @@ public class Mekanism extends ModPropertyContainer {
     public final ThermalEvaporationPlant thermalEvaporationPlant = new ThermalEvaporationPlant();
     public final Washer washer = new Washer();
 
-    public Mekanism() {
-        addRegistry(infusion);
-
-        addRegistry(chemicalInfuser);
-        addRegistry(combiner);
-        addRegistry(crusher);
-        addRegistry(crystallizer);
-        addRegistry(dissolutionChamber);
-        addRegistry(electrolyticSeparator);
-        addRegistry(enrichmentChamber);
-        addRegistry(injectionChamber);
-        addRegistry(metallurgicInfuser);
-        addRegistry(osmiumCompressor);
-        addRegistry(chemicalOxidizer);
-        addRegistry(pressurizedReactionChamber);
-        addRegistry(purificationChamber);
-        addRegistry(sawmill);
-        addRegistry(smelting);
-        addRegistry(solarNeutronActivator);
-        addRegistry(thermalEvaporationPlant);
-        addRegistry(washer);
-    }
-
     @Override
-    public void initialize() {
-        GameObjectHandler.builder("gas", GasStack.class)
-                .mod("mekanism")
+    public void initialize(GroovyContainer<?> container) {
+        container.objectMapper("gas", GasStack.class)
                 .parser((s, args) -> {
                     Gas gas = GasRegistry.getGas(s);
                     return gas == null ? Result.error() : Result.some(new GasStack(gas, 1));
                 })
                 .completerOfNamed(GasRegistry::getRegisteredGasses, Gas::getName)
+                .docOfType("gas stack")
                 .register();
-        GameObjectHandler.builder("infusion", InfuseType.class)
-                .mod("mekanism")
+        container.objectMapper("infusionType", InfuseType.class) // infusion clashes with infusion field
```

Comment:
`mods.mekanism.infusion` is already a virtualized registry so we cant use `mods.mekanism.infusion()` for mapper

Reply:
it currently seems to work in the scripts without erroring, accessing both `mods.mekanism.infusion('thing')` and `mods.mekanism.infusion.add(...)`, when testing locally with a renamed object mapper. this seems to contradict what you are saying. could you clarify?

how viable is changing object mappers to use a different location - eg `objectMapper.[mod].[name]` instead of putting it inside `mods`?

if this cant be done, these are a few other object mappers that need to be renamed - i havent looked through all of them, but thaumcraft's `aspect` definitely has a collision.